Add tiddly-wiki-server to TiddlyWeb implementors

nathanielknight opened the pull request:
Add [tiddly-wiki-server](https://github.com/nathanielknight/tiddly-wiki-server) to the [list of TiddlyWeb](https://tiddlywiki.com/#TiddlyWeb) implementors.

Tiddly-Wiki server uses SQLite as a backing datastore and is implemented in Rust. It's designed to have a much smaller memory footprint than the existing NodeJS based back-end; as one might expect, it compromises on some other features.

Jermolene commented:
Hi @nathanielknight, and welcome to the project. I am sorry I missed this last week. I hadn't seen tiddly-wiki-server before, and it's great to see both Rust and SQLite.

With apologies for the bureaucracy, please could you prepare a separate PR to the 'tiddlywiki-com' branch with your signature for the Contributor License Agreement (see [contributing.md](https://github.com/Jermolene/TiddlyWiki5/blob/master/contributing.md)).

nathanielknight commented:
Hello! Thanks for the generous welcome. ☺️

I've added my signature, and updated this PR to conform to the requirements laid out in the contributor's guide.

Full disclosure, I'm the only contributor to `tiddly-wiki-server`; it's stood up to all my testing, so I'd like to share it a little more broadly in case someone has a use case I haven't covered.

Jermolene commented:
Thanks @nathanielknight
